base/Makefile

```diff
@@ -55,7 +55,7 @@ else
 endif
 	@echo "const libfftw_name = \"$(LIBFFTWNAME)\"" >> $@
 	@echo "const libfftwf_name = \"$(LIBFFTWFNAME)\"" >> $@
-	@echo "const libllvm_version = \"$$($(LLVM_CONFIG_HOST) --version)\"" >> $@
+	@echo "const libllvm_version_string = \"$$($(LLVM_CONFIG_HOST) --version)\"" >> $@
```

Comment:
Yeah I agree, but there isn't a way to do this without making unnecessary changes/shuffling of files.   `VERSION_STRING` follows the same strategy as in this PR.

Any opinions on the uppercasing (in regards to consistency) ?